.github/workflows/main.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     name: Build
     strategy:
       matrix:
-        node-version: [12, 13, 14]
+        node-version: [14, 15, 16, 17, 18, 19]

```

Comment:
Let's just support LTS versions, I think we should drop `15`, `17` and `19` as they go EOL as soon another release is launched.